### ISSUE TITLE
UI on mobile device

### ISSUE THREAD

**Issue by FreddieM007** [stevenwaterman/musetree#81]
It would be great if the tool were also available on mobile devices. I tried MuseTree on an Android phone: The UI opens and responds and looks like that it should work but there does not seem to be a way to right-mouse click nodes. I tried double taps and holding them for several seconds to no avail.
Unless this is a user error, it would be great to enable right-click on mobile devices.

**Comment by stevenwaterman** [comment]
I've put no effort into making it mobile-friendly, and can confirm that it doesn't work at all. There's a lot more problems than just not being able to right-click, and I will try and fix them eventually, but it's going to be a lower-priority issue.

**Comment by FreddieM007** [comment]
Yes, I agree. It is not really important at this stage. Once everything is said and done, perhaps there is an opportunity to look into it.
